src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+        for (int i = 0; i < 6; i++) {
+            if (i < 3) {
+                supplier.getRandomFigure().draw();
+                continue;
+            }
+            supplier.getDefaultFigure().draw();
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. In this case, although you're not creating a new `FigureSupplier` in each iteration, it's important to ensure that the `FigureSupplier` instance is not being recreated unnecessarily within the method calls themselves.